### ISSUE TITLE
mean-variance normalization

### ISSUE THREAD

**Issue by Pepslee** [BVLC/caffe#4674]
What is difference between the mvn(mean-variance normalization) layer and the batchNorm layer ???

**Comment by shelhamer** [comment]
Batch norm can calculate windowed statistics/moving averages and switch between accumulating or using fixed statistics. `MVN` layer simply centers and standardizes a single batch a at a time.
